examples/ota-provider-app/esp32/main/BdxOtaSender.cpp

```diff
@@ -174,7 +174,6 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
         {
             ChipLogError(BDX, "onTransferComplete Callback not set");
         }
-        mStopPolling = true; // Stop polling the TransferSession only after receiving BlockAckEOF
         Reset();
```

Comment:
so we removed the mStopPolling flag being set here but how would we notify the transfer session to stop polling. I do not see a path where we call https://github.com/project-chip/connectedhomeip/pull/34538/files#diff-98b0f488c3e61ca466c5c9c383db17e6b66f4784505b8d18fd90c9d4c172f4cbR40 which stops the timer in the transfer facilitator. The Reset() here only calls Reset on the transfer object and that doesn't cancel the timer. I see that the destructor for TransferFacilitator calls the ResetTransfer API that stops the timer. so are we assuming the transfer facilitator object will be destroyed? What am I missing here?

Reply:
`Reset()` here is the [BdxOtaSender::Reset](https://github.com/project-chip/connectedhomeip/blob/693dd386e8681984d8856fb114738f6dba2c11bd/examples/ota-provider-app/esp32/main/BdxOtaSender.cpp#L226). And it calls `ResetTransfer` inside